projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -47,11 +47,12 @@ import { TableWidgetModel } from './table-widget.model';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <ht-titled-content
+      class="table-title"
```

Comment:
I see there's already a `[ngClass]="{ 'header-margin': this.model.header?.topMargin }"` - is that the same thing? If the two don't have different purposes, can we resolve to one way of doing it?

Reply:
The header margin is for the table controls header. The new one is for the title that sits above that.

Reply:
<img width="445" alt="Screen Shot 2021-03-24 at 11 07 30 AM" src="https://user-images.githubusercontent.com/45181984/112361946-21ca8880-8c91-11eb-87dd-f292dd972d25.png">

Reply:
I think that's a bug though - the header model where this top margin is defined is used for the title, not table controls. I'm guessing when table controls were added, it created an unintentional separation between the title and where that margin was applied. If you look where this is used today, it's only in one spot - backend overview, on a table that doesn't have controls (which is likely why the bug was never caught - it was still rendering the same).

Reply:
So the places where the header is used without topMargin being set just look bad - they should have this property set. If we want to remove that property entirely and automatically add the appropriate padding if title is set, I think that works:

![image](https://user-images.githubusercontent.com/45047841/112364941-4e33d400-8c94-11eb-9e68-50a715b30a5f.png)

We just want to make sure we're not double padding it if that property is set, so we should do one or the other.